### PR TITLE
Missing (epochs_)tmin and tmax

### PR DESCRIPTION
**Pull request by crsegerie**
There was some missing (epochs_) in front tmin and tmax params in some config files.

### Before merging …
- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

**Comment by crsegerie** [comment]
I don't know if those commit in the test folder will change the documentation here https://mne.tools/mne-bids-pipeline/examples/ds003392.html

**Review by agramfort — APPROVED** [review]
awesome @crsegerie !

🙏 

you'll need to fix merge conflict in changes.md file though

File: docs/source/changes.md [hunk 1]
```diff
@@ -67,3 +67,5 @@ authors:
   ({{ gh(320) }} by {{ authors.hoechenberger }})
 - The sanity check comparing the rank of the experimental data and the rank of the empty-room after Maxwell-filtering did not use the maxfiltered data
   (by {{ authors.agramfort }}, {{ authors.hoechenberger }}, and {{ crsegerie }}).
+- Missing (epochs_)tmin and (epochs_)tmax in some config files.
+  (by crsegerie)
```
Comment (hoechenberger) [inline docs/source/changes.md:71]:
```suggestion
- The sanity check comparing the rank of the experimental data and the rank of the empty-room after Maxwell-filtering did not use the maxfiltered data.
  (by {{ authors.agramfort }}, {{ authors.hoechenberger }}, and {{authors.crsegerie }})
- `epochs_tmin` and `epochs_tmax` were named incorrectly in some config files.
  (by {{authors.crsegerie }})
```

**Review by hoechenberger — COMMENTED** [review]
Otherwise LGTM!

**Comment by hoechenberger** [comment]
Thanks @crsegerie!

**Pull request merged**